container-search/src/main/java/com/yahoo/search/query/UniqueRequestId.java

```diff
@@ -0,0 +1,28 @@
+package com.yahoo.search.query;
+
+import com.yahoo.container.Server;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class UniqueRequestId {
```

Comment:
Not unless I move it up to .search package, and that felt worse as there are only Query, Result and Searcher.
Could it be in a non exported package ?